.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: NPM publish on release 
+on: 
+  push: 
+    tags: 
+      - v[0-9].[0-9].[0-9]
+
+jobs: 
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    with:
+      node-version: 16
+      registry-url: 'https://registry.npmjs.org'
+    - run: npm install
+    - run: |
+        npm prepack
+        npm publish 
+      env: 
+        GITHUB_WORKSPACE: ${GITHUB_WORKSPACE}
```

Comment:
@toniya-klaytn Why do you need this one?

Reply:
npm prepack is to build the library